src/dotnet/CarbonAware.WebApi/Program.cs

```diff
@@ -10,7 +9,7 @@
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
 
-builder.Services.AddCarbonAwareServices();
+builder.Services.AddEmissionServices(PluginType.JSON);
```

Comment:
It would be good to add here the registration with the `IAggregator` instead of using the plugin directly.
Basically adding a ConfiguratioServiceExtension to the Aggregator, it would register itself and the ICarbonAware Service.

Reply:
Pushed the change, let me know if that's what you had in mind? I didn't know if we wanted to do it with the IAggregator specifically since we could have different aggregators (I.e., a carbon aware one, an energy one etc.) have the same functionality

Reply:
@JenMadiedo , I like what you have, but seems to miss the registration of `ICarbonAwareAggregator` with `CarbonAwareAggregator`, on the extension of the aggregator.

Reply:
Added!